keras_hub/src/models/sam/sam_image_segmenter.py

```diff
@@ -4,6 +4,9 @@
 from keras_hub.src.api_export import keras_hub_export
 from keras_hub.src.models.image_segmenter import ImageSegmenter
 from keras_hub.src.models.sam.sam_backbone import SAMBackbone
+from keras_hub.src.models.sam.sam_image_segmenter_preprocessor import (
+    SamImageSegmenterPreprocessor,
```

Comment:
Why do we call `SAMBackbone` and `SamImageSegmenter`? Should we use ALL upper case or all mixed case?

Reply:
Thanks for the review Ramesh! you are right, I have updated the name to match the convention in the erst of the dir.